flowauth/backend/flowauth/__init__.py

```diff
@@ -54,13 +55,15 @@ def create_app(test_config=None):
     csrf = CSRFProtect()
     csrf.init_app(app)
     csrf.exempt(login_blueprint)
+    csrf.exempt(version_blueprint)
```

Comment:
Just for my own understanding: why do these blueprints need to be exempt from CSRF protection?

Reply:
Can't guarantee you have the csrf cookie when you hit them.